include/base/ghosting_functor.h

```diff
@@ -130,6 +130,20 @@ class Elem;
  * (e.g. subdomain-restricted variables on a neighboring subdomain)
  * will be unaffected.
  *
+ * Typical usage of the GhostingFunctor would be to add the appropriate
+ * functor before EquationSystems::init() is called. However, in some
```

Comment:
This is actually misleading - algebraic and coupling functors should be added before EquationSystems::init(), but geometric ghosting functors are ideally added before the Mesh is filled.

Reply:
> geometric ghosting functors are ideally added before the Mesh is filled.

Meaning before `prepare_for_use()`? Sorry, I'm only messing with the algebraic/coupling functors so I'm less sure about the geometry functors.

Reply:
That's the right meaning, but I'm not sure of a better way to say it than "filled", since libMesh mesh generation idioms typically do prepare_for_use() implicitly.